### PR TITLE
goarch param for orchestrator

### PR DESCRIPTION
**Pull request by rcerven**
Signed-off-by: Robert Cerven <rcerven@redhat.com>

### CODE REVIEW

**Comment by twaugh** [comment]
Restarted the failing test.

File: osbs/build/build_request.py [hunk 1]
```diff
@@ -317,6 +317,11 @@ def render_orchestrate_build(self):
 
         self.dj.dock_json_set_arg(phase, plugin, 'config_kwargs', config_kwargs)
 
+        goarch = {}
+        for platform in self.platform_descriptors:
```
[the remark this answers is not in the document]
Comment (rcerven) [reply osbs/build/build_request.py]: ok

**Comment by twaugh** [comment]
Test run timed out again. I think it's because of all the combinations of inputs to this test. Is there anything that can be done to improve test performance?

File: osbs/build/build_request.py [hunk 2]
```diff
@@ -317,6 +317,11 @@ def render_orchestrate_build(self):
 
         self.dj.dock_json_set_arg(phase, plugin, 'config_kwargs', config_kwargs)
 
+        goarch = {}
+        for platform, architecture in self.platform_descriptors.items():
+            goarch[platform] = architecture
+        self.dj.dock_json_set_arg(phase, plugin, 'goarch', goarch)
+
```
Comment (lcarva) [inline osbs/build/build_request.py]: Also need to set up OpenShift params: `url=None, verify_ssl=True, use_auth=True,`

Comment (rcerven) [reply osbs/build/build_request.py]: right will add, thanks

**Comment by rcerven** [comment]
combined some combinations in test_render_orchestrator, so it is faster now

**Comment by lcarva** [comment]
Updated release notes draft:
> - `orchestrate_build` plugin is rendered with goarch and openshift parameters

**Pull request merged**